explainer.md

```diff
@@ -315,28 +336,173 @@ an author could specify this relationship programmatically instead:
 ```js
 const input = comboBox.shadowRoot.querySelector("input");
 const optionList = comboBox.querySelector("custom-optionlist");
-input.accessibleNode.activeDescendant = optionList;
+input.accessibleNode.activeDescendant = optionList.accessibleNode;
 ```
 
 This would allow the relationship to be expressed naturally.
 
+#### No reflection
+
+Even though there's correspondence between Accessible Properties and
+ARIA attributes, they are never reflected. Similarly, Accessible Properties
+do not reflect equivalent implicit properties of their associated DOM elements,
+either.
+
+```js
+button = document.createElement("button");
+button.innerHTML = "Next";
+console.log(button.accessibleNode.role);   // null, not "button"
```

Comment:
My concern here is that this example concerns the computed properties rather than the ARIA properties. I think we should keep those concerns completely separate - this section should only talk about ARIA properties, and the discussion of computed properties should come in the "Computed accessible properties" section which (now) follows immediately after. Make sense?